core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -102,43 +102,35 @@ public static Snapshot oldestAncestor(Table table) {
   }
 
   /**
-   * Traverses the history of the table's current snapshot and:
-   * 1. returns null, if no snapshot exists or target timestamp is more recent than the current snapshot.
-   * 2. else return the first snapshot which satisfies {@literal >=} targetTimestamp.
-   * <p>
-   * Given the snapshots (with timestamp): [S1 (10), S2 (11), S3 (12), S4 (14)]
-   * <p>
-   * firstSnapshotAfterTimestamp(table, x {@literal <=} 10) = S1
-   * firstSnapshotAfterTimestamp(table, 11) = S2
-   * firstSnapshotAfterTimestamp(table, 13) = S4
-   * firstSnapshotAfterTimestamp(table, 14) = S4
-   * firstSnapshotAfterTimestamp(table, x {@literal >} 14) = null
-   * <p>
-   * where x is the target timestamp in milliseconds and Si is the snapshot
+   * Traverses the history of the table's current snapshot and finds the first snapshot after the given timestamp.
```

Comment:
"First snapshot created after" I was confused about this description before because "after" could refer to the traversal which would be the snapshot one older than the timestamp rather than the  snapshot one younger than the timestamp which I believe is the intent here

Reply:
I see what you mean. We need to specify what we're ordering by. I'll update this.